samples/bluetooth/radio_notification_cb/sample.yaml

```diff
@@ -9,12 +9,10 @@ tests:
       - nrf52dk/nrf52832
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpunet
-      - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
There is no nrf54l15dk/nrf54l15/cpuapp on the list. It this expected?

Reply:
@rugeGerritsen ^^

Reply:
It should be added